python/paddle/python_wrapper_demo/variable.py

```diff
@@ -0,0 +1,67 @@
+from namespace import Namespace
+from common import g_scope
+
+
+class Variable(object):
+    '''
+    Variable is the data type of Operator's inputs and outputs.
+    '''
+    counter = 0
+    __varset__ = set()
+
+    def __init__(self,
+                 name=None,
+                 shape=[],
+                 data=None,
+                 initialzier=None,
+                 scope=g_scope,
```

Comment:
In python, complex instance cannot be default value.

Reply:
```python
In [1]: class A(object):
   ...:     def __init__(self):
   ...:         self.x = None
   ...:         

In [2]: def f(x=A()):
   ...:     print x
   ...:     

In [3]: f()
<__main__.A object at 0x10c7cc050>
```

this seems works

Reply:
See http://effbot.org/zone/default-values.htm